edge-util/src/Microsoft.Azure.Devices.Edge.Util.Metrics/Microsoft.Azure.Devices.Edge.Util.Metrics.csproj

```diff
@@ -20,11 +20,14 @@
     <PackageReference Include="prometheus-net.AspNetCore" Version="4.2.0" />
 
     <!--
-      prometheus-net.AspNetCore takes a transitive dependency on
-      System.Test.Encodings.Web. Use an explicit reference here to override the
-      version and fix a security vulnerability.
+      prometheus-net.AspNetCore takes transitive dependencies on
```

Comment:
can we update the version of prometheus-net.AspNetCore? Or does the latest version also not reference the latest AspNetCore libs?

Reply:
4.2.0 was the last of the 4.x line, the current version is 7.0 (released just a couple weeks ago). It's not clear to me if they're strictly following semantic versioning or not. The readme for 7.0 claims to have a few breaking changes, but at a glance I don't think they'll affect our code.